sites/platform/src/integrations/notifications.md

```diff
@@ -74,17 +74,42 @@ You can also configure a custom `--from-address`. The `--from-address` is whatev
 
 ### Slack notifications
 
-A notification can trigger a message to be sent to a Slack bot.
-First, create a new custom "[bot user](https://api.slack.com/bot-users)" for your Slack group and configure the channels you wish it to live in.
-Note the API token is the "Bot User OAuth Access Token" provided by Slack.
-
-Then register that Slack bot with {{% vendor/name %}} using a `health.slack` integration:
-
-```bash
-{{% vendor/cli %}} integration:add --type health.slack --token YOUR_API_TOKEN --channel '#channelname'
-```
-
-That will trigger the corresponding bot to post a notification to the `#channelname` channel in your Slack group.
+A notification can trigger a message to be sent to a [Slack app](https://api.slack.com/apps).
+
+1. [Create a Slack app](https://api.slack.com/apps), if you haven't done so already.
+    Choose whether to build the app from scratch, or via [an app manifest](https://api.slack.com/concepts/manifests).
+    Give the app a name and pick a workspace to develop your app in. 
+    Note that if you are not an admin of the workspace, you will need to request approval to install it there later.
+1. Once created, go to **OAuth & Permissions** under **Features** in the sidebar.
+1. Towards the bottom of the page there is a card titled **Scopes**.
+    Under **User Token Scopes**, **Add an OAuth Scope** - specifically the `chat:write` scope.
+
+    ![Slack app scopes](/images/slack/slack-app-scopes.png "0.30")
+
+1. Under **Settings** in the sidebar select **Install app**. 
+    Add this point, if you are _not_ an admin of the workspace, you will need to provide a link to actually install the app into the workspace and onto a channel.
+    Once the app is approved and installed, a **User OAuth Token** will be provided in your app settings. 
+    Copy the token, and use in the next step. 
+1. Using the **User OAuth Token**, run the command
+
+    ```bash
+    {{% vendor/cli %}} integration:add --type health.slack --token {{% variable "USER_OAUTH_TOKEN" %}} --channel {{% variable "CHANNEL_NAME" %}} --project {{% variable "PROJECT_ID" %}}
+    ```
+
+    If the channel you're installing the app is called `project-notifications`, write the channel name in the command above as you would reference it within Slack. 
+    That is, 
+
+    ```bash
+    {{% vendor/cli %}} integration:add --type health.slack ... --channel '#project-notifications' ...
+    ```
+1. {{% vendor/name %}} will then send an initial message to the channel once the integration is successfully configured. 
```

Comment:
```suggestion
{{% note theme="info" %}}
If you are _not_ a workspace admin, but an admin has approved your app for installation it may first be necessary to install the application to the desired channel within Slack prior to running the above command. 
{{% /note %}}


1. {{% vendor/name %}} will then send an initial message to the channel once the integration is successfully configured. 
```